_v1.1/docs/user-guide/working-with-resources.md

```diff
@@ -59,7 +59,7 @@ The API will generally not modify fields that you have set; it just sets ones wh
 
 ## <a name="finding_schema_docs"></a>Finding Documentation on Resource Fields
 
-You can browse auto-generated API documentation at the [project website](http://kubernetes.io/v1.0/api-ref.html) or on [github](https://releases.k8s.io//docs/api-reference).
+You can browse auto-generated API documentation at the [project website](http://kubernetes.io/v1.0/api-ref.html) or on [github](https://releases.k8s.io/v1.1.0/docs/api-reference).
```

Comment:
This line is not cherry-picked to release-1.1. https://github.com/kubernetes/kubernetes/pull/16043/files#r42919463